src/resources/Organizations/OrganizationInterfaces.ts

```diff
@@ -16,22 +16,31 @@ export interface OrganizationsStatusModel {
     supportActivated: boolean;
 }
 
-export interface OrganizationModel {
+export type OrganizationModel = {
     id: string;
     displayName: string;
     createdDate: number;
-    license: LicenseModel;
-    organizationStatusModel: OrganizationsStatusModel;
     publicContentOnly: boolean;
     type: string;
     owner: {
         email: string;
     };
     readOnly: boolean;
+} & AdditionalOrganizationFieldsModel;
+
+export interface AdditionalOrganizationFieldsModel {
+    license?: LicenseModel;
+    organizationStatusModel?: OrganizationsStatusModel;
+}
+
+export type AdditionalOrganizationField = 'status' | 'license' | string;
+
+export interface GetOrganizationOptions {
+    additionalFields?: AdditionalOrganizationField | AdditionalOrganizationField[];
 }
 
 export interface ListOrganizationOptions {
-    additionalFields?: string | string[];
+    additionalFields?: AdditionalOrganizationField | AdditionalOrganizationField[];
```

Comment:
Can we extend GetOrganizationOptions in ListOrganizationOptions instead of additionalFields * 2?

Reply:
I am not completely against this suggestion, but I usually like to keep `extends` when it actually "extends" something 😄 

I feel like, over time, `GET` could have options that would not be available on `LIST`, so I would rather duplicate one line and have future implementers ask themselves whether it should be in both GET and LIST than assuming they will always have both options.